src/v2/Components/ArtworkFilter/ArtworkFilters/index.tsx

```diff
@@ -28,23 +27,21 @@ interface ArtworkFiltersProps {
 // Some filters will be rendered only if there is the necessary data in aggregations (for example, ArtistsFilter)
 export const ArtworkFilters: React.FC<ArtworkFiltersProps> = props => {
   const { user, relayEnvironment } = props
-  const {
-    contextPageOwnerId,
-    contextPageOwnerSlug,
-    contextPageOwnerType,
-  } = useAnalyticsContext()
-  const { match } = useRouter()
+  let currentPage
 
-  const isArtistPage = match.location.pathname.slice(1, 7) === OwnerType.artist
+  if (typeof window !== "undefined") {
+    const { pageType } = getContextPageFromClient()
+    currentPage = pageType
+  }
```

Comment:
If we're going to leave this here then I'd at least add a `TODO` or `HACK` comment regarding the why of it and maybe a note to switch to checking the route.

Mocking useRouter is easy to do however if that's really the blocker.

Reply:
@mc-jones exactly! Also this is the [sample unleash doc](https://github.com/artsy/force/blob/main/docs/unleash_a-b_testing.md) file that I added that we can update with our learnings etc